apps/frontend/src/components/card/CardDetails.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styled from 'styled-components';
+import InfoProjectBadges from '../badges/InfoProjectBadges';
+import CardContent from '../content/CardContent';
+
+interface CardContentProps {
+  titleCard: string;
+  infoStatus?: string;
+  desc: string;
+  adminInitial: string;
```

Comment:
These props shouldn't be re-declared here but you should reuse those of the `ProjectBadges` component instead.
> see `intersection-types` https://www.typescriptlang.org/docs/handbook/unions-and-intersections.html#intersection-types

Reply:
What do you think if declare my props on the `ProjectCard` component ? 
They include the props of the `InfoProjectBadges`. I don't have any errors but is this the right way to do it?